web_app_drawer/views/assets.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    @author Dave Lasley <dave@laslabs.com>
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+    <template id="assets_backend" name="Open Mobile Assets" inherit_id="web.assets_backend">
+        <xpath expr=".">
+            <link rel="stylesheet"
+                  type="text/css"
+                  href="/web_app_drawer/static/lib/css/drawer.3.2.0.css"
+                  />
+            <link rel="stylesheet"
+                  href="/web_app_drawer/static/src/less/variables.less"
```

Comment:
Weird I thought the variables would need to be declared first. Leave it to UI ppl to come up with some backwardsness like this!